ployst/github/test/test_end_to_end.py

```diff
@@ -15,7 +15,7 @@ class TestEndToEnd(TestCase):
     def setUp(self):
         ensure_dummy_clone_available()
 
-    @patch(__name__ + '.tasks.hierarchy.client', MockClient())
+    @patch('ployst.github.tasks.hierarchy.client', MockClient())
```

Comment:
I import ``tasks`` at the top of the module.

http://stackoverflow.com/questions/14341689/how-to-mock-using-patch-relative-paths